File: @LV1/1.3/pot/pot.cpp
#include <cmath>
#include <cstdio>

int main() {
    int n, tmp, sum = 0;
    scanf("%d", &n);

    for (int i = 0; i < n; i++) {
        scanf("%d", &tmp);
        sum += pow(tmp / 10, tmp % 10);
    }
    printf("%d\n", sum);
}

File: @LV1/1.3/pot/pot.go
package main

import "fmt"

func pow(a int, b int) int {
	ans := 1
	for i := 0; i < b; i++ {
		ans *= a
	}
	return ans
}

func main() {
	var n, tmp, sum int

	fmt.Scanf("%d", &n)

	for i := 0; i < n; i++ {
		fmt.Scanf("%d", &tmp)
		sum += pow(tmp/10, tmp%10)
	}
	fmt.Println(sum)
}
